lambda/common/src/main/scala/com/gu/hq/lambda/model/events.scala

```diff
@@ -44,3 +44,8 @@ case class Relationship(
   resourceName: Option[String],
   name: String
 )
+
+case class AccountMap(
```

Comment:
Not sure this is in the right place?

This is nothing to do with events, so why not here?

https://github.com/guardian/security-hq/blob/master/lambda/common/src/main/scala/com/gu/hq/lambda/model/aws.scala

Reply:
Moved to new file accounts.scala.  It didn't really fit with anything else either.